src/Models/File.php

```diff
@@ -43,7 +43,9 @@ public function toCmsArray()
             'original' => FileService::getUrl($this->uuid),
             'size' => $this->size,
             'filesizeInMb' => number_format($this->attributes['size'] / 1048576, 2),
-        ];
+        ] + (config('twill.file-library.show_uploaded_date')
+            ? ['uploadedDate' => $this->created_at->format("d/m/Y H:i")]
```

Comment:
Seems this is not using the newly added option `twill.file-library.format_uploaded_date`

Reply:
@gilbo75 do you need this feature on a Twill 2 project. If you do, please address the issue mentioned by Tofandel and I can merge and release a new minor on Twill 2.

Reply:
Hello,
Yes, I need this feature on Twill 2